app/ui/src/main/java/com/fsck/k9/activity/EditIdentity.kt

```diff
@@ -0,0 +1,98 @@
+package com.fsck.k9.activity
+
+import android.os.Bundle
+import android.view.View
+import com.fsck.k9.Account
+import com.fsck.k9.Identity
+import com.fsck.k9.Preferences
+import com.fsck.k9.ui.R
+import kotlinx.android.synthetic.main.edit_identity.*
+
+class EditIdentity : K9Activity() {
+    private lateinit var mAccount: Account
```

Comment:
Ideally, we'd do a (minimal) cleanup in a separate commit right after the conversion to Kotlin. The use of hungarian notation in Kotlin makes me cringe :smile: